File: src/main/java/com/github/penguin418/todolist/repository/TodoUserRepository.java
package com.github.penguin418.todolist.repository;

import com.github.penguin418.todolist.model.entity.TodoUserEntity;
import org.springframework.data.repository.CrudRepository;

import java.util.Optional;

public interface TodoUserRepository extends CrudRepository<TodoUserEntity, Long> {
    Optional<TodoUserEntity> findByUsername(String username);


}

File: src/main/java/com/github/penguin418/todolist/repository/TodoUserRepository.kt
package com.github.penguin418.todolist.repository

import com.github.penguin418.todolist.model.entity.TodoUserEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.*
@Repository
interface TodoUserRepository : CrudRepository<TodoUserEntity, Long> {
    fun findByUsername(username: String): Optional<TodoUserEntity>
}
